CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# For contributors
+ 1. Please have fun, nobody is paid for developing Panopticon.
+ 1. Everything is up to debate, none of the following guidelines is set in stone.
+ 1. I you want to help, check out the [easy issues](https://github.com/das-labor/panopticon/issues?q=is%3Aissue+is%3Aopen+label%3AE-easy).
+ 1. If you want a feature, open an issue _before_ you're dumping your 5k line patch.
+ 1. Format w/ rust-fmt before submitting.
+ 1. Please add tests, at least one positive and one negative test. Make sure the coverage doesn't decrease.
+ 1. Consider splitting your change into multiple smaller PRs.
+ 1. All CI jobs must pass. You have an infinite amount of tries.
+
+# For maintainers & reviewers
+ 1. Assume good faith.
+ 1. Please have fun, nobody is paid for developing Panopticon.
+ 1. Everything is up to debate, none of the following guidelines is set in stone.
+ 1. Check whenever contributed code:
```

Comment:
As my etymology professor liked to say: "English is orthographically conservative" :grin: